### PR TITLE
Edit sigmoid_2d_plot.ipynb figure.

### PR DESCRIPTION
**Pull request by rohit-khoiwal-30**
### Figure Number
Book1 - 10.2

### Figures
| Before PR | fig 10 2 |
| :-: | :-: |
| After PR | fig10 2_latexified |

## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

### CODE REVIEW

File: notebooks/book1/10/sigmoid_2d_plot.ipynb [hunk 1]
```diff
@@ -0,0 +1,211 @@
+{
```
Comment (nipunbatra) [inline notebooks/book1/10/sigmoid_2d_plot.ipynb:1]: <p>Some initial comments</p><p> </p><ul><li>What do the colours mean?</li><li>We can replace numpy with jax.numpy</li><li>We can use jax.nn.sigmoid directly, etc. </li><li>The overall x-axis and y-axis in the original plot are labelled and show the coordinates. Why have we removed them?</li><li>The caption linked in the PR does not use LaTeX. Symbols like &gt; should be replaced with $\gt$, etc.</li></ul><p>cc @karm-patel</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (karm-patel) [reply notebooks/book1/10/sigmoid_2d_plot.ipynb:1]: Prof @nipunbatra, Related to the last comment about Latex, We are just copy-pasting the caption from pdf to save time because we are adding caption for just completeness, and also, a caption will be rendered from latex source code only. Please let me know if you want to suggest something to this.

Comment (karm-patel) [reply notebooks/book1/10/sigmoid_2d_plot.ipynb:1]: @rohit-khoiwal-30 Have you addressed the comments

Comment (nipunbatra) [reply notebooks/book1/10/sigmoid_2d_plot.ipynb:1]: > Prof @nipunbatra, Related to the last comment about Latex, We are just copy-pasting the caption from pdf to save time because we are adding caption for just completeness, and also, a caption will be rendered from latex source code only. Please let me know if you want to suggest something to this.

Ok. No problem,
